src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -74,10 +74,6 @@ limitations under the License.
                 <requireAttribute>ACTIVE_LOCALE</requireAttribute>
                 <requireAttribute>SUPPORTED_LOCALES</requireAttribute>
             </include>
-            <include cluster="Time Format Localization" client="false" server="true" clientLocked="true" serverLocked="false">
```

Comment:
Why is this being removed?

Reply:
The Cluster shall indeed stay listed under root endpoint but it is not mandatory so only change server="true" to server="false" and not require any attribute should do what you are looking for. Same for Unit Localization.